tox.ini

```
@@ -1,29 +1,29 @@
 [tox]
 envlist =
     # Python/Django combinations that are officially supported
-    py{27,35,36,37}-django111
-    py{35,36,37,38}-django22
-    py{36,37,38}-django30
+    py{36,37,38,39}-django22
+    py{36,37,38,39}-django30
+    py{36,37,38,39}-django31
```

You could consolidate those 3 lines into a single one, e.g.
```suggestion
    py{36,37,38,39}-django{22,30,31}
```